src/domain/spaceAdmin/layout/SpaceAdminTabsL1.tsx

```diff
@@ -22,7 +22,7 @@ export const spaceAdminTabsL1: TabDefinition<SettingsSection>[] = [
     icon: ForumOutlinedIcon,
   },
   {
-    section: SettingsSection.Subsubspaces,
+    section: SettingsSection.Subspaces,
```

Comment:
I think with this we can finally remove Subsubspaes from the SettingsSection enum